Spanish/01_Day_Introduction/helloworld.py

```diff
@@ -0,0 +1,22 @@
+# Introducción
+# Día 1 - Desafío 30DaysOfPython
+
+print(2 + 3)   # suma(+)
+print(3 - 1)   # resta(-)
+print(2 * 3)   # multiplicación(*)
+print(3 / 2)   # división(/)
+print(3 ** 2)  # exponencial(**)
+print(3 % 2)   # módulo(%)
+print(3 // 2)  # División de piso(//)
```

Comment:
I think "División entera" conveys the meaning better (I have never heard "Division de piso" and Google results show things related to construction):

```suggestion
print(3 // 2)  # División entera (//)
```